Taskfile.yml

```diff
@@ -76,7 +67,7 @@ tasks:
       - "webui"
       - "webui-nodejs"
     cmds:
-      - task: "clean-package"
+      - "rm -rf '{{.OUTPUT_DIR}}'"
```

Comment:
Just thinking out loud, we have talked about having a package where task overwrites and rsync it to another package where we actually run start_clp to avoid removing the database and compressed archive. Any chance we can add this flow to the task file?

Anyway, it doesn't need to be in this PR

Reply:
Yeah, I thought about it a bit, but just haven't done any work on it. I'll add a task for it.